client/src/main/com/sinch/sdk/domains/sms/models/v1/batches/response/Batch.java

```diff
@@ -0,0 +1,7 @@
+package com.sinch.sdk.domains.sms.models.v1.batches.response;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.sinch.sdk.domains.sms.models.v1.batches.response.internal.BatchResponseInternalImpl;
+
+@JsonDeserialize(using = BatchResponseInternalImpl.Deserializer.class)
+public interface Batch {}
```

Comment:
As discussed, can you double-check the naming: `Batch` vs `BatchResponse` ?

Reply:
Right.
Back to generator to change naming -> WIP

Reply:
Updated